content/asset-tracking/example-apps.textile

```diff
@@ -63,6 +63,43 @@ blang[android].
     <img src="/images/screenshots/asset-tracking/android/example-apps-running.png" style="width: 100%" alt="Example apps running">
   </a>
 
+blang[swift].
+  h2(#using). Using the example apps
+
+  1. Clone the Ably Asset Tracking SDK Swift "GitHub repo":https://github.com/ably/ably-asset-tracking-swift.
+  2. Open @AblyAssetTracking.xcworkspace@ in Xcode. "This workspace includes a Swift package containing the SDKs, and several example apps":https://github.com/ably/ably-asset-tracking-swift#package-structure.
+  3. Configure your @~/.netrc@ file with your Mapbox secret access token that enables you to download the Mapbox SDK. You can find instructions on how to do this "in the Mapbox docs":https://docs.mapbox.com/ios/search/guides/install/#configure-credentials.
+  4. Copy the @Examples/Example.Secrets.xcconfig@ file to @Example/Secrets.xcconfig@.
+  5. In @Example/Secrets.xcconfig@, configure your @ABLY_API_KEY@ and @MAPBOX_ACCESS_TOKEN@. This is the public Mapbox token, which you can find on your Mapbox account's "token page":https://account.mapbox.com/access-tokens/.
+  6. Select the @PublisherExample@ project, choose a target device, and then press the run button.
+
+  <a href="/images/screenshots/asset-tracking/swift/run-publishing-example-app.png" target="_blank">
+    <img src="/images/screenshots/asset-tracking/swift/run-publishing-example-app.png" style="width: 100%" alt="Run the publishing app">
+  </a>
+
+  The following screen displays:
+
+  <a href="/images/screenshots/asset-tracking/swift/publisher-enter-tracking-id.png" target="_blank">
+    <img src="/images/screenshots/asset-tracking/swift/publisher-enter-tracking-id.png" style="width: 50%" alt="Provide a tracking ID">
+  </a>
+
+  7. Click the "Ask for location permissions" button and grant the appropriate permissions.
```

Comment:
Small consistency point: in the Android instructions button presses aren't in quotations. We probably need to agree a format for this and add it to the style guide.

Reply:
Good point. Removed for now to be like the Android version.